homeassistant/components/sensor/rmvtransport.py

```diff
@@ -0,0 +1,210 @@
+"""
+Support for real-time departure information for Rhein-Main public transport.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.rmvdeparture/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, ATTR_ATTRIBUTION, STATE_UNKNOWN
+    )
+
+REQUIREMENTS = ['PyRMVtransport==0.0.6']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_NEXT_DEPARTURE = 'nextdeparture'
+
+CONF_STATION = 'station'
+CONF_DESTINATIONS = 'destinations'
+CONF_DIRECTIONS = 'directions'
+CONF_LINES = 'lines'
+CONF_PRODUCTS = 'products'
+CONF_TIMEOFFSET = 'timeoffset'
+CONF_MAXJOURNEYS = 'max'
+
+DEFAULT_NAME = 'RMV Journey'
+
+VALID_PRODUCTS = ['U-Bahn', 'Tram', 'Bus', 'S', 'RB', 'RE', 'EC', 'IC', 'ICE']
+
+ICONS = {
+    'U-Bahn': 'mdi:subway',
+    'Tram': 'mdi:tram',
+    'Bus': 'mdi:bus',
+    'S': 'mdi:train',
+    'RB': 'mdi:train',
+    'RE': 'mdi:train',
+    'EC': 'mdi:train',
+    'IC': 'mdi:train',
+    'ICE': 'mdi:train',
+    'SEV': 'mdi:checkbox-blank-circle-outline',
+    None: 'mdi:clock'
+}
+ATTRIBUTION = "Data provided by opendata.rmv.de"
+
+SCAN_INTERVAL = timedelta(seconds=30)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_NEXT_DEPARTURE): [{
+        vol.Required(CONF_STATION): cv.string,
+        vol.Optional(CONF_DESTINATIONS, default=['']): cv.ensure_list_csv,
+        vol.Optional(CONF_DIRECTIONS, default=['']): cv.ensure_list_csv,
+        vol.Optional(CONF_LINES, default=['']): cv.ensure_list_csv,
+        vol.Optional(CONF_PRODUCTS, default=VALID_PRODUCTS):
+            vol.All(cv.ensure_list, [vol.In(VALID_PRODUCTS)]),
+        vol.Optional(CONF_TIMEOFFSET, default=0): cv.positive_int,
+        vol.Optional(CONF_MAXJOURNEYS, default=5): cv.positive_int,
```

Comment:
What do we call this in other sensor commuting platforms?

Reply:
`max`/`CONF_MAXJOURNEYS`: *MVG* calles it `number`, but I found that a bit unspecific. It is the only I found which has a similar setting.

`timeoffset`/`CONF_TIMEOFFSET`: *MVG* calles it `timeoffset` as well, *Public Transit (GTFS)* calles it `offset` and *Västtrafik Public Transport* calles it `delay`. A bit of a mix.

Reply:
Ok. `max` isn't so specific either I think. But I don't have a better suggestion at the moment.

It would be good to come up with a descriptive name and then have all platforms use the same term. But it can wait to a different PR.

But please separate words by underscore in variable and constant names and strings. So change to `CONF_TIME_OFFSET = 'time_offset'` and `CONF_MAX_JOURNEYS`.